lab_solutions/lab02_14_l2_disctance.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Attila Nagy <attila.nagy234@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+
+def l2_distance(vector1, vector2):
+    dim1 = len(vector1)
+    dim2 = len(vector2)
+    dim_diff = abs(dim1 - dim2)
+    if dim_diff != 0:
+        raise ValueError(f'The dimensions of the provided two vectors are not equal, so cannot compute their L2 distance. v1_dim: {dim1}, v2_dim: {dim2}')
```

Comment:
You don't need such a long error message, people don't read long messages. 'Vectors dimensions must be equal' would be enough.